src/osd/PrimaryLogPG.cc

```diff
@@ -1646,6 +1646,7 @@ void PrimaryLogPG::do_request(
 	     << p->first << " not empty, queueing" << dendl;
     p->second.push_back(op);
     op->mark_delayed("waiting_for_map not empty");
+    osd->request_osdmap_update(op->min_epoch);
```

Comment:
@jdurgin why not request_osdmap_update at the beginning when op must wait for map?
https://github.com/ceph/ceph/pull/17795/files#diff-fb41013d27e932534adb50eb3de2aaa5R1656

Reply:
argh that's where it's meant to be, must have moved this around by accident... fixing